### DISCUSSION TITLE
Install Extension that not compatible with 115.10.0

### DISCUSSION THREAD

**Discussion by Demo337** [Floorp-Projects/Floorp#1094] [Feature Request]
I need to install Enhancer for Youtube extension in floorp but I couldn't because it not compatible with 115.10.0.
I know this is not a  Feature Request nor a reporting a bug but I think this is the best place to submit the issue. 

![Screenshot 2024-04-15 135639](https://github.com/Floorp-Projects/Floorp/assets/36269127/06a0a4ef-d4bb-45dd-a202-0e04dc4997a7)

**Comment by creeper-0910** [comment]
Enabling "Do not check for compatibility with add-ons" should solve the problem.

**Reply by Demo337** [reply]
It worked, thanks

**Reply by noahnao** [reply]
How do you do that, I cannot, for the life of me, find that setting anywhere!

**Reply by breadbeforerice** [reply]
If you scroll down to the bottom of about:preferences, is it not at the top? It is under Enable Legacy Components

**Reply by Ryder17z** [reply]
> If you scroll down to the bottom of about:preferences, is it not at the top? It is under Enable Legacy Components

it is very much worth mentioning that the mozilla addons page will still complain about the browser version and that you have to click "download file" to install it (making the wording feel wrong)

**Reply by creeper-0910** [reply]
In that case, you may need to disguise the UserAgent as the latest version of Firefox.
